src/main/java/seedu/partyplanet/logic/commands/ToggleThemeCommand.java

```diff
@@ -0,0 +1,32 @@
+package seedu.partyplanet.logic.commands;
+
+import seedu.partyplanet.model.Model;
+import seedu.partyplanet.ui.Theme;
+
+/**
+ * Toggles the theme of PartyPlanet between Dark and Pink.
```

Comment:
Missed out in initial suggestions
```suggestion
 * Toggles the theme of PartyPlanet between Dark and Pastel.
```